src/pan3d/utils/common.py

```diff
@@ -419,6 +444,120 @@ def save_dataset(self, file_path):
         self.state.show_save_dialog = False
         return asynchronous.create_task(self._save_dataset(file_path))
 
+    def _create_standard_ui(
```

Comment:
I don't like that... You should be able to use the a base pan3d layout class where you can fill the available slots.